pkg/sfu/streamtrackermanager.go

```diff
@@ -33,16 +75,22 @@ func (s *StreamTrackerManager) OnAvailableLayersChanged(f func(availableLayers [
 }
 
 func (s *StreamTrackerManager) AddTracker(layer int32) {
-	cycleDuration := 500 * time.Millisecond
-	samplesRequired := uint32(5)
-	cyclesRequired := uint64(60) // 30s of continuous stream
-	if layer == 0 {
-		// be very forgiving for base layer to account for cases like static screen share where there could be only one packet per second
-		samplesRequired = 1
-		cyclesRequired = 1 // 1 packet in 2 seconds
-		cycleDuration = 2 * time.Second
-	}
-	tracker := NewStreamTracker(s.logger, samplesRequired, cyclesRequired, cycleDuration)
+	var params StreamTrackerParams
+	if s.source == livekit.TrackSource_SCREEN_SHARE || s.source == livekit.TrackSource_SCREEN_SHARE_AUDIO {
```

Comment:
screen share audio wouldn't go through here? since only video is subject to stream tracker

Reply:
Oh, is that the audio track captured with screen share? I thought it was a special attribution for screen share when captured with audio included. Then, I need to remove that check in a couple of places. Let me do that.

Reply:
Made relevant changes in this commit - https://github.com/livekit/livekit-server/pull/503/commits/1df74622537a6be80eb72201398c344dfe1fad35

Reply:
yup, that's it! 👍